src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function AddSpanToUl(root) {
+  const ul = root.querySelectorAll('li');
```

Comment:
Why do you select `li` elements and name the const `ul`? Maybe `listItems` would be better